Shared/Samples/Display device location with NMEA data sources/README.md

```diff
@@ -0,0 +1,76 @@
+# Display device location with NMEA data sources
```

Comment:
After tomorrow's meeting we might need to change this sample's name (Display -> Show). I'll let you know when the change happens.

Reply:
Please see `common-samples/pull/3862` for the details. To rename a sample, these places need to be looked at

- the folder name
- README title
- title in metadata
- `.swift` filename
- view type name
- screenshot caption in README and png filename